lib/miq_memcached.rb

```diff
@@ -2,6 +2,10 @@
 require 'linux_admin'
 
 module MiqMemcached
+  def self.server_address
+    ENV["MEMCACHED_SERVER"] || ::Settings.session.memcache_server
```

Comment:
If this is just for containers should we use the `ENV["CONTAINER"]` here and just switch this directly to using `ENV["MEMCACHED_SERVICE_NAME"]` which is the variable this is already exposed as in the container?

Or even just `s/SERVER/SERVICE_NAME/` and assume it won't be there in any other environment?